gh-user-activity

```diff
@@ -17,4 +18,7 @@ TEMPLATE='
   {{- end -}}
 '
 
-gh api -H 'accept: application/vnd.github.v3+json' "users/$GH_USER/events?per_page=100" --template="${TEMPLATE}"
+for a in $(eval echo "{1..$GH_PAGES}")
+do
+  gh api -H 'accept: application/vnd.github.v3+json' "users/$GH_USER/events?per_page=100&page=$a" --template="${TEMPLATE}" || echo ok
```

Comment:
My concern is that the fallback to `ok` would swallow any failure in the `gh` command and will print non-expected output to stdout. 
What do you think about removing the fallback ?

Reply:
Yes echo ok was put while fiddling with the pagination settings, I thought that the API command was randomly failing hence that echo ok. It can be removed sorry.

Reply:
Removed echo ok